webrtc/videoroom/assets/src/room.ts

```diff
@@ -101,19 +100,22 @@ const setup = async () => {
           if (isScreenSharing) {
             setScreensharing(stream, displayName || "", "My screensharing");
           } else {
-            addAudioElement(stream);
+            addVideoElement(stream, displayName || "", false);
```

Comment:
ahhh, I think that I forgot to clean things up when introducing participat labeling. `getDisplayNamOrRedirect` no longer redirect anywhere, it just takes display name from query params and cleans up the url, can you please change the function name and then replace `displayName || ""` with `displayName`?